### PR TITLE
Support 'start' attribute for ordered list

### PR DESCRIPTION
**Pull request by pjona**

### CODE REVIEW

**Comment by pjona** [comment]
is something wrong with this?

File: _class/parsingCss.class.php [hunk 1]
```diff
@@ -1022,6 +1022,10 @@ public function analyse($tagName, &$param, $legacy = null)
                     $this->value[$nom] = $val;
                     break;
 
+                case 'start':
+                    $this->value[$nom] = $val;
+                break;
```
Comment (juchi) [inline _class/parsingCss.class.php]: incorrect indent for break

File: html2pdf.class.php [hunk 2]
```diff
@@ -1182,8 +1182,12 @@ protected function _listeAddLevel($type = 'ul', $style = '', $img = null)
             else                $style = 'decimal';
         }
 
+        if ($start <= 0)
+            $start = 1;
+        $nb_start = $start - 1;
```
Comment (juchi) [inline html2pdf.class.php]: please use camelCase for variable name

File: html2pdf.class.php [hunk 3]
```diff
@@ -4446,7 +4450,7 @@ protected function _tag_open_UL($param, $other = 'ul')
         $this->_tag_open_TABLE($param, $other);
 
         // add a level of list
-        $this->_listeAddLevel($other, $this->parsingCss->value['list-style-type'], $this->parsingCss->value['list-style-image']);
+        $this->_listeAddLevel($other, $this->parsingCss->value['list-style-type'], $this->parsingCss->value['list-style-image'], $this->parsingCss->value['start']);
```
Comment (juchi) [inline html2pdf.class.php]: if a 'start' property is not set in the CSS, this index won't be defined. You need to set a default value in initStyle().

File: _class/parsingCss.class.php (hunk 1 above)
Comment (spipu) [inline _class/parsingCss.class.php]: and the value $val must be converted in (int)

File: html2pdf.class.php (hunk 2 above)
Comment (spipu) [inline html2pdf.class.php]:
```
if (is_null($start) || (int) $start<1) {
    $start=0;
} else {
    $start--;
}
```

File: html2pdf.class.php (hunk 3 above)
Comment (spipu) [inline html2pdf.class.php]: or you can prepare the value before : 
`$start = (isset($this->parsingCss->value['start']) ? $this->parsingCss->value['start'] : null);`

**Comment by pjona** [comment]
Thanks for the reply!
I will send fixes today

**Comment by pjona** [comment]
I sent fixes

**Comment by spipu** [comment]
@todo: report this feature in the master branch

**Comment by pjona** [comment]
@spipu: you'll want to release a new version with this fix? I can add information to ChangeLog and move this fix to master.

**Comment by spipu** [comment]
yes, it could be cool :)
add it to the changelog, and try to add it in the master. Be carefull, all the classes have move, you can not merge your branch on the master

**Pull request merged**